e2e-network/k8s/test-01-simple-k8s.sh

```diff
@@ -60,22 +56,23 @@ waitForContainer "$peer0" "Membership view has changed. peers went online:"
 waitForContainer "$peer1" "Learning about the configured anchor peers of Org1MSP for channel my-channel1"
 waitForContainer "$peer1" "Membership view has changed. peers went online:"
 
+
 #Test simple chaincode
-expectInvoke "admin" "org1-peer1.default" "chaincode1" "my-channel1" \
-  "put -a '[\"name\"]' -a Willy Wonka" "{\"success\":\"OK\"}"
-expectInvoke "admin" "org1-peer1.default" "chaincode1" "my-channel1" \
-  "get -a '[\"name\"]'" '{"success":"Willy"}'
+expectInvoke "admin" "org1-peer1.default" "my-channel1" "chaincode1" \
```

Comment:
peer domain names should come from fablo-config file - does it work this way? Do we have `org1-peer1.default` in fablo-config file for this test?

Reply:
Took a look at the fablo-config file, peer domain names are not found. What i see:

```   
"peer": {
        "instances": 2,
        "db": "LevelDb"
},
```

In org1.yaml
```
peers:
  org1-peer0.default:
    grpcOptions:
      allow-insecure: false
      hostnameOverride: ""
      ssl-target-name-override: ""
    tlsCACerts:
      pem: |
    url: grpcs://172.18.0.2:31734
  org1-peer1.default:
    grpcOptions:
      allow-insecure: false
      hostnameOverride: ""
      ssl-target-name-override: ""
    tlsCACerts:
      pem: |
    url: grpcs://172.18.0.2:31824
```

Reply:
have a look how domain names are created for docker configuration: https://github.com/umegbewe/fablo/blob/k8/src/setup-docker/templates/fabric-docker/docker-compose.yaml

we have for instance `<%= peer.address %>` or `<%= peer.fullAddress %>` available. If it is possible, it should do it the same way for kubernetes (we should take values from `<%= peer.address %>` in our scripts)

Reply:
I'm quite interested how we can do this in a bash script, to use this template values i think it should be at time of generation,  not sure we are generating bash scripts.

for e.g
expectInvoke "admin" "<%= peer.address %>" "my-channel1" "chaincode1"?